pkg/operator/oauth.go

```diff
@@ -62,6 +64,31 @@ func ApplyOAuth(cr *v1alpha1.Console, rt *routev1.Route) (*oauthv1.OAuthClient,
 	return authClient, authConfigSecret, nil
 }
 
+func DeleteOAuthSecret(cr *v1alpha1.Console) error {
```

Comment:
Remove.

Reply:
I do create this secret & own it. 
I do not own the client.

Reply:
Sorry, side effect of reviewing too quickly.